datadog_lambda/wrapper.py

```diff
@@ -138,14 +149,24 @@ def _before(self, event, context):
         except Exception:
             traceback.print_exc()
 
-    def _after(self, event, context):
+    def _after(self, event, context, response, trigger_tags):
         try:
+            status_code = extract_http_status_code_tag(trigger_tags, response)
+            if status_code:
+                trigger_tags["http.status_code"] = status_code
+            # Create a Datadog subsegment with trigger tags
```

Comment:
I don't think the future reader will be able to understand what really does or why it's needed from this comment.